tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +112,167 @@ module('Acceptance | guid file | registration files', hooks => {
             .hasAria('label', t('file_detail.view_revisions'), 'Versions button has correct label when closed');
     });
 
-    // test for tags
+    // Test file metadata
+    test('view metadata', async function(this: ThisTestContext, assert) {
+        const nodeNoun = ['Registration', 'Project', 'Component'];
+        const metadataRecord = await this.owner.lookup('service:store')
+            .findRecord('custom-item-metadata-record', this.registration.id);
+
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        // Verify correct default tab for dekstop
+        assert.dom('[data-test-metadata-tab]').exists();
```

Comment:
This test selector does not exist, so you have a few failing tests.

Reply:
Ok, this has been added.